specification/oracle/Oracle.Database/tspconfig.yaml

```diff
@@ -22,10 +22,6 @@ options:
     namespace: com.azure.resourcemanager.oracledatabase
     partial-update: true
     flavor: azure
-  "@azure-tools/typespec-python":
-    package-dir: "azure-mgmt-oracledatabase"
-    package-name: "{package-dir}"
-    flavor: azure
```

Comment:
please resolve the conflict with main branch @ChenxiJiang333

Reply:
updated, please help rereview @msyyc